intel_extension_for_transformers/transformers/modeling/modeling_auto.py

```diff
@@ -342,42 +286,45 @@ def from_pretrained(cls, pretrained_model_name_or_path, *model_args, **kwargs):
                 model.quantization_config = quantization_config
                 return model
             else:
-                kwargs["low_cpu_mem_usage"] = True
-                kwargs["device_map"] = "cpu"
-                try:
-                    model = cls.ORIG_MODEL.from_pretrained(
-                        pretrained_model_name_or_path,
-                        torchscript=True
-                        if quantization_config.algorithm in ["TEQ", "AWQ"] and not use_xpu
-                        else False,
-                        *model_args,
-                        **kwargs,
-                    )
-                    model.config.update({"low_cpu_mem_usage": True})
-                except NotImplementedError:
-                    logger.info("Failed to load models with `low_cpu_mem_usage` specified, "
-                                "will fall to traditional load method with higher memory consumption.")
-                    kwargs["low_cpu_mem_usage"] = False
+                if use_xpu:
+                    kwargs["low_cpu_mem_usage"] = True
+                    kwargs["device_map"] = "auto"

```

Comment:
please find the real root cause

Reply:
it is gptj issue, if we set low_cpu_mem_usage"=True, then the ACC is incorrect.